definition/primitives.js

```diff
@@ -36,34 +36,24 @@ module.exports = {
       }
     ]
   },
-
-  // TODO - extract and test
   recps: {
     oneOf: [
       { type: 'null' },
       {
         type: 'array',
         items: {
-          oneOf: [ // TODO - remove this (write tests first)
-            { $ref: '#/definitions/feedId' }
-            // { $ref: '#/definitions/mentions/feed' }
+          oneOf: [
+            { $ref: '#/definitions/feedId' },
+            { $ref: '#/definitions/mentions/feed' }
```

Comment:
@christianbundy now that we've established that Patchbay has been publishing odd private messages and that it's patchcore that has been encrypting them ... I think we should decide what the deal is with this recps definition.

Personally I still lean toward removing this `mentions/feed` thing. I think if a person want this to be a definition for some schema, that's fine, but it's non-standard and they can roll their own `recps` definition just fine.

YEAH. drop this.. is what I vote for

Reply:
Cool, I can remove this. Is that the only blocker on this PR?